### PR TITLE
Implement models

### PR DESCRIPTION
**Pull request by DzmitrySalanovich**

### CODE REVIEW

File: authentication/validators.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import datetime
+from django.core import validators
+from django.core.exceptions import ValidationError
+from django.utils.deconstruct import deconstructible
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_birthday(birthday):
+    if birthday > datetime.date.today():
+        raise ValidationError("The birthdat can't be in future")
```
Comment (MaksimLion) [inline authentication/validators.py]: again typo

File: products/models.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    quantity = models.IntegerField()
+    category = models.ManyToManyField(to=Category)
+
+    def __str__(self):
+        return self.name
+        
+class Product_att(models.Model):
+    name = models.CharField(max_length=30)
+    value = models.DecimalField(max_digits=10, decimal_places=3)
+    product_id = models.ForeignKey(Product, on_delete=models.CASCADE)
```
Comment (MaksimLion) [inline products/models.py]: I think you should add related_name to ForeignKey field

File: products/models.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    price = models.DecimalField(max_digits=10, decimal_places=3)
+    quantity = models.IntegerField()
+    category = models.ManyToManyField(to=Category)
+
+    def __str__(self):
+        return self.name
+        
+class Product_att(models.Model):
+    name = models.CharField(max_length=30)
+    value = models.DecimalField(max_digits=10, decimal_places=3)
```
Comment (MaksimLion) [inline products/models.py]: Why Decimal Field, If I want to add Color attribute to Product, How I can do it?

File: authentication/validators.py (hunk 1 above)
Comment (duckyou) [inline authentication/validators.py]: Just telling that there is a typo :wink:

File: products/models.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    def __str__(self):
+        return self.name
+
+class Product(models.Model):
+    name = models.CharField(max_length=30)
+    sku = models.IntegerField()
+    discount = models.DecimalField(max_digits=10, decimal_places=3)
+    price = models.DecimalField(max_digits=10, decimal_places=3)
+    quantity = models.IntegerField()
```
Comment (MaksimLion) [inline products/models.py]: You can use PositiveIntegerField, otherwise you should add validators

File: orders/models.py [hunk 5]
```diff
@@ -1,3 +1,28 @@
...
+    user = models.ForeignKey(to=User, on_delete = models.CASCADE)
+    cost = models.DecimalField(max_digits=10, decimal_places=3)
+    cc_number = models.CharField(max_length=15)
+    status = models.CharField(choices=STATUSES, max_length=100)
+
+
+class LineItem(models.Model):
+    product_sku = models.IntegerField()
+    order_id = models.ForeignKey(Order, on_delete=models.CASCADE)
+    datatime = models.DateTimeField('date ordered')
```
Comment (MaksimLion) [inline orders/models.py]: Could you make the date automatically?

**Review by duckyou — APPROVED** [review]

File: config/settings.py [hunk 6]
```diff
@@ -38,6 +38,10 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'authentication',
+    'config',
```
Comment (MaksimLion) [inline config/settings.py]: Why you added config to installed apps?

**Review by MaksimLion — CHANGES_REQUESTED** [review]

File: products/models.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    quantity = models.PositiveIntegerField()
+    category = models.ManyToManyField(to=Category)
+
+    def __str__(self):
+        return self.name
+
+class Product_att(models.Model):
+    name = models.CharField(max_length=30)
+    value = models.CharField(max_length=10)
+    product_id = models.ForeignKey(Product, on_delete=models.CASCADE, related_name='attribute')
```
Comment (MaksimLion) [inline products/models.py:26]: attributes

File: orders/models.py [hunk 8]
```diff
@@ -1,3 +1,28 @@
...
+class Order(models.Model):
+    AVAILABLE = 'AV'
+    IN_PROGRESS = 'INPR'
+    COMPLITE = 'CMP'
+    STATUSES = [
+        (AVAILABLE, 'Available'),
+        (IN_PROGRESS, 'In progress'),
+        (COMPLITE, 'Complite'),
+    ]
+    user = models.ForeignKey(to=User, on_delete = models.CASCADE)
```
Comment (MaksimLion) [inline orders/models.py:15]: add related_name

**Review by MaksimLion — APPROVED** [review]

**Pull request merged**